code/game/objects/structures.dm

```diff
@@ -73,9 +73,16 @@
 
 	usr.visible_message("<span class='warning'>[user] starts climbing onto \the [src]!</span>")
 	climber = user
-	if(!do_after(user, 50, target = src))
-		climber = null
-		return FALSE
+	if(climber.mind.assigned_role == "Bartender" || climber.mind.assigned_role == "Chef")
```

Comment:
Could you make this a trait instead that is added in `/datum/outfit/job/bartender/post_equip`, with the source being a new trait source called `JOB_TRAIT` (this is what I'd prefer), if you're lazy, `ROUNDSTART_TRAIT` works as well

Reply:
Given I only want this on roundstart/latejoin jobs, i think ill just toss it on `ROUNDSTART_TRAIT`